Assets/Scripts/Audio/FishAudio.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.Serialization;
+
+public class FishAudio : MonoBehaviour
+{
+    [Header("Audio Clips")]
+    [SerializeField] private AudioClip[] swimClips;
+    [SerializeField] private AudioClip[] dashClips;
+    [SerializeField] private AudioClip fallingClip;
```

Comment:
Can a fish fall? 😂

Reply:
Is that a philosophical question?
Lol jokes aside, I left that there just so I remember how to add `OneShot` audio clips. I'll remove it for now